client/disputes/index.tsx

```diff
@@ -74,33 +79,39 @@ const headers: DisputesTableHeader[] = [
 		required: true,
 	},
 	{
-		key: 'customer',
+		key: 'customerName',
 		label: __( 'Customer', 'woocommerce-payments' ),
 		isLeftAligned: true,
 	},
 	{
-		key: 'email',
+		key: 'customerEmail',
 		label: __( 'Email', 'woocommerce-payments' ),
 		visible: false,
 		isLeftAligned: true,
 	},
 	{
-		key: 'country',
+		key: 'customerCountry',
 		label: __( 'Country', 'woocommerce-payments' ),
 		visible: false,
 		isLeftAligned: true,
 	},
 	{
 		key: 'created',
 		label: __( 'Disputed on', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Disputed on', 'woocommerce-payments' ),
 		required: true,
 		isLeftAligned: true,
+		isSortable: true,
+		defaultSort: true,
+		defaultOrder: 'desc',
 	},
 	{
 		key: 'dueBy',
 		label: __( 'Respond by', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Respond by', 'woocommerce-payments' ),
```

Comment:
Why do we add `screenReaderLabel` here (and other entries)?

Reply:
I was not using it initially but I noticed that the snapshots showed 'Sort by **undefined** in ascending order' in the markup related to screen readers. I found that using the `screenReaderLabel ` the name of the label renders correctly. This is how it looks in Jest if I omit the `screenReaderLabel`:

![Screen Shot 2022-01-20 at 10 29 26](https://user-images.githubusercontent.com/1553182/150312359-a4d1ff15-3ec0-4129-ae49-e1894e2f52c8.png)

I believe it is fine to use this, but please let me know if it is not the case.

Reply:
I see, then let's add those for all headers here.

Reply:
Done!